scipy/stats/tests/test_distributions.py

```diff
@@ -3867,6 +3867,16 @@ def test_fit(self):
         a7m, loc7m, scale7m = stats.skewnorm.fit(-rvs, method='mm')
         assert_allclose([a7m, loc7m, scale7m], [-a7p, -loc7p, scale7p])
 
+        # test that MLE is resistant to finding the wrong local optimum (see gh-19332)
+        a_correct, loc_correct, scale_correct = -1.60957, 2.2990, 2.46210
```

Comment:
How are these known to be correct? What is the source of this test case?

Reply:
The source of this test case is unfortunately rather involved. I have a somewhat hacky Mathematica notebook where I compute MLE with `NMaximize` and `WorkingPrecision->100`. I evaluate this for `n=0` and `n=7` to get an error interval and truncate the numbers to the last digit of agreement.

As for evidence that this is the correct global optimum I also have some Python code where I verify that the likelihood from the skewnorm fit is better than the corresponding normal and half-normal best fits.

Reply:
If you want to give evidence that the optimum is global, you could use `scipy.stats.fit`? Not required by me, though.

Reply:
Ah, so if I understand what you're saying here: fit generically misses, but can be helped along by hand by providing these bounds?

Should I do n=0...7 or just n=0?

Reply:
No, [`scipy.stats.fit`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.fit.html) is a function that uses global optimization (differential evolution, by default) to do the fitting. It needs bounds to work, so it is not used by `scipy.stats.rv_continuous.fit` (or `fit` of subclasses).
 
Any single case (e.g. `n=1`) is enough, since it would fail in main and (presumably) pass in this PR.